packages/trellix_edr_cloud/data_stream/event/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,941 @@
+---
+description: Pipeline for processing Event logs.
+processors:
+  - set:
+      field: ecs.version
+      tag: set_ecs_version
+      value: 8.9.0
```

```suggestion
      value: 8.8.0
```
Since its set to `8.8.0` in `build.yml`

---

Hey @kcreddy,
As 8.9.0 is released so we can update the ecs version in build.yml as 8.9.0.

---

Correct. please do that.